hector.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: index
+title: Climate Module – Hector	
+gcam-version: v4.2
+---
+
+This section describes the new climate module - Hector - that is available for use in GCAM. In GCAM, in addition to running MAGICC 5.3 (Wigley, 2008), there is the option to run Hector (Hartin et al., 2015).  Both Hector and MAGICC are reduced-form climate carbon-cycle models. 
+
+Hector, an open-source, object-oriented, reduced-form global climate carbon-cycle model, written in C++. This model runs essentially instantaneously while still representing the most critical global-scale earth system processes. Hector has a three-part main carbon cycle: a one-pool atmosphere, land, and ocean. The model’s terrestrial carbon cycle includes primary production and respiration fluxes, accommodating arbitrary geographic divisions into, e.g., ecological biomes or political units. Hector actively solves the inorganic carbon system in the surface ocean, directly calculating air– sea fluxes of carbon and ocean pH. Hector reproduces the global historical trends of atmospheric [CO<sub>2</sub>], radiative forcing, and surface temperatures. The model simulates all four Representative Concentration Pathways (RCPs) with equivalent rates of change of key variables over time compared to current observations, MAGICC (a well-known simple climate model), and models from the 5th Coupled Model Intercomparison Project. Hector’s flexibility, open-source nature, and modular design will facilitate a broad range of research in various areas. 
```

Comment:
"...is written in C++"
Remove "(a well-known simple climate model)"
Maybe replace "5th Coupled Model Intercomparison Project" with "[CMIP5](http://cmip-pcmdi.llnl.gov/cmip5/)"
Replace "will facilitate" with "facilitates"